src/python/pants/backend/python/rules/coverage.py

```diff
@@ -127,20 +144,39 @@ class CoverageConfig:
     digest: Digest
 
 
+def _validate_and_update_config(coverage_config: configparser.ConfigParser) -> None:
+    if not coverage_config.has_section("run"):
+        coverage_config.add_section("run")
+        coverage_config.set("run", "branch", "True")
+    run_section = coverage_config["run"]
+    relative_files_str = run_section.get("relative_files", "True")
+    if relative_files_str.lower() != "true":
+        raise ValueError("relative_files under the 'run' section must be set to True")
+    coverage_config.set("run", "relative_files", "True")
+    omit_elements = [em.strip() for em in run_section.get("omit", "").split("\n") if em.strip()]
+    if "test_runner.pex/*" not in omit_elements:
+        omit_elements.append("test_runner.pex/*")
```

Comment:
We might want to prefix this with `\n`. The example in the docs shows them starting on a new line: https://coverage.readthedocs.io/en/coverage-5.2/source.html#execution.